data-raw/vs_peds.R

```diff
@@ -2,160 +2,218 @@
+# Convert blank to NA ----
+# vs <- convert_blanks_to_na(vs)
+
+# Subset to 5 patients present in dm_peds ----
+vs_subset <- vs %>%
+  filter(USUBJID %in% c(
+    "01-701-1015", "01-701-1023", "01-701-1028",
+    "01-701-1033", "01-701-1034"
+  ) &
+    VSTESTCD %in% c("WEIGHT"))
+
+# Filter 'WEIGHT' records for placeholder for HEIGHT and HDCIRC
+vs_subset_height <- vs_subset %>%
+  filter(VSTESTCD == "WEIGHT") %>%
+  mutate(VSTESTCD = "HEIGHT")
+vs_subset_bmi <- vs_subset %>%
+  filter(VSTESTCD == "WEIGHT") %>%
+  mutate(VSTESTCD = "BMI")
+vs_subset_hdcirc <- vs_subset %>%
+  filter(VSTESTCD == "WEIGHT") %>%
+  mutate(VSTESTCD = "HDCIRC")
+
+# Bind new HEIGHT records to original dataset
+vs_subset_full <- bind_rows(
+  vs_subset,
+  vs_subset_height,
+  vs_subset_bmi,
+  vs_subset_hdcirc
 ) %>%
+  arrange(USUBJID, VISITNUM, VSTESTCD, VSDY)
+
+# Updating subject 01-701-1015 with data for 1 year old girl
+vs_peds <- vs_subset_full %>%
+  mutate(VSSTRESN = case_when(
+    USUBJID == "01-701-1015" & VSTESTCD == "WEIGHT" ~ case_when(
+      VISIT == "SCREENING 1" ~ 9.11,
+      VISIT == "BASELINE" ~ 9.2,
+      VISIT == "WEEK 2" ~ 9.32,
+      VISIT == "WEEK 4" ~ 9.54,
+      VISIT == "WEEK 6" ~ 9.7,
+      VISIT == "WEEK 8" ~ 9.91,
+      VISIT == "WEEK 12" ~ 10.3,
+      VISIT == "WEEK 16" ~ 10.7,
+      VISIT == "WEEK 20" ~ 11.1,
+      VISIT == "WEEK 24" ~ 11.56,
+      VISIT == "WEEK 26" ~ 11.71,
+    ),
+    USUBJID == "01-701-1015" & VSTESTCD == "HEIGHT" ~ case_when(
+      VISIT == "SCREENING 1" ~ 74.13,
+      VISIT == "BASELINE" ~ 74.41,
+      VISIT == "WEEK 2" ~ 74.71,
+      VISIT == "WEEK 4" ~ 75.32,
+      VISIT == "WEEK 6" ~ 75.93,
+      VISIT == "WEEK 8" ~ 76.54,
+      VISIT == "WEEK 12" ~ 77.72,
+      VISIT == "WEEK 16" ~ 78.96,
+      VISIT == "WEEK 20" ~ 80.22,
+      VISIT == "WEEK 24" ~ 81.43,
+      VISIT == "WEEK 26" ~ 82.03
+    ),
+    USUBJID == "01-701-1015" & VSTESTCD == "HDCIRC" ~ case_when(
+      VISIT == "SCREENING 1" ~ 35.61,
+      VISIT == "BASELINE" ~ 37.24,
+      VISIT == "WEEK 2" ~ 38.57,
+      VISIT == "WEEK 4" ~ 39.84,
+      VISIT == "WEEK 6" ~ 40.95,
+      VISIT == "WEEK 8" ~ 42.14,
+      VISIT == "WEEK 12" ~ 43.67,
+      VISIT == "WEEK 16" ~ 44.79,
+      VISIT == "WEEK 20" ~ 45.92,
+      VISIT == "WEEK 24" ~ 46.88,
+      VISIT == "WEEK 26" ~ 47.56
+    ),
+    USUBJID == "01-701-1023" & VSTESTCD == "WEIGHT" ~ case_when(
+      VISIT == "SCREENING 1" ~ 12.47,
+      VISIT == "BASELINE" ~ 12.89,
+      VISIT == "WEEK 2" ~ 13.14,
+      VISIT == "WEEK 4" ~ 13.45
+    ),
+    USUBJID == "01-701-1023" & VSTESTCD == "HEIGHT" ~ case_when(
+      VISIT == "SCREENING 1" ~ 87.65,
+      VISIT == "BASELINE" ~ 88.25,
+      VISIT == "WEEK 2" ~ 88.75,
+      VISIT == "WEEK 4" ~ 89.23
+    ),
+    USUBJID == "01-701-1023" & VSTESTCD == "HDCIRC" ~ case_when(
+      VISIT == "SCREENING 1" ~ 87.65,
+      VISIT == "BASELINE" ~ 88.25,
+      VISIT == "WEEK 2" ~ 88.75,
+      VISIT == "WEEK 4" ~ 89.23
```

Comment:
Some of the head circumferences seem extremely high - i would make them maximum 50 as per the medians from https://github.com/pharmaverse/admiralpeds/blob/main/data-raw/who_hc_for_age_boys.R

Reply:
Thank you for the finding. That was my copy-paste mistake. I copied the section with height values, then updated the pram name and missed updating the values. The issue has been corrected.